index.html

```diff
@@ -1,19 +1,71 @@
 <!doctype html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-  <title>My Resume</title>
+    <title>Alicia Petgrave | Resume</title>
 
-  <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="css/index.css">
+    
 
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
-</head>
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
 
-<body>
-  <h1>My Resume</h1>
-</body>
+  <body>
+    <header>
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">About</a>
+        <a href="https://aliciapetgrave.github.io" target=_blank>Portfolio</a>
+        <a href="#">Contact</a>
+      </nav>
+    </header>
+
+    <div class="bodycontainer">
+      <!--INTRODUCTION-->
+      <div class="intro">
+        <section>
+          <h1>Alicia Petgrave | Full Stack Web Developer</h1>
+          <p>Hi! I'm a full stack web developer who focuses on building with the user in mind. I'm passionate about learning and using my knowledge to create beautifully functioning web apps. I am a hard-working, self disciplined person who is very easy to work with and eager to create.</p>
+        </section>
+      </div>
+
+      <div class="contact">
+        <p>Fort Lauderdale, FL</p>
+        <p>123-456-7890</p>
+        <p>Alicia.Petgrave@gmail.com</p>
+      </div>
+
+
+      <!--SKILLS-->
+      <div class="main1">
+        <section class="skillwork">
+          <h2>Skills</h2> 
+          
+          <p>HTML</p><br>
+          <p>CSS</p><br>
+          <p>Git</p><br>
+          <p>Less</p><br>
+          <p>Javascript</p>
+        </section>
+        
+
+        <!--WORK HISTORY-->
+        
+        <section class="skillwork">
+          <h2>Employment History</h2>
+          <li>Lambda School 2019 - Present</li><br>
```

Comment:
Since you listed these as  list items but really only using the li tag for the bullet point, you can put these in p tags and enter in 
& #8226; or & bull
for the • Circular Bullet Point